src/encoded/schemas/atac_reproducibility_quality_metric.json

```diff
@@ -0,0 +1,124 @@
+{
+    "title": "ATAC-seq reproducibility quality metric",
+    "description": "Schema for reporting ATAC-seq reproducibility quality metrics",
+    "id": "/profiles/atac_reproducibility_quality_metric.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["step_run","quality_metric_of", "award", "lab", "reproducibility_type"],
+    "additionalProperties": false,
+    "identifyingProperties": ["uuid"],
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "quality_metric.json#/properties" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/submitted"},
+        { "$ref": "mixins.json#/aliases"},
+        { "$ref": "mixins.json#/attribution" },
+        { "$ref": "mixins.json#/standard_status"},
+        { "$ref": "mixins.json#/assay" }
+    ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "reproducibility_type": {
+            "title": "Reproducibility type",
+            "type": "string",
+            "enum": [
+                "overlap",
+                "IDR"
+            ]
+        },
+        "Nx": {
+            "title": "Reproducible peaks",
+            "description": "Number of peaks called from one replicate or pooled replicates passing reproducibility test by comparing self-pseudoreplicates or pooled pseudo-replicates. The single peak file is associated with this quality metric.",
+            "type": "integer",
+            "minimum": 0
+        },
+        "N_pair": {
+            "title": "Reproducible peaks between 2 replicates",
+            "description": "Number of peaks passing reproducibility test between 2 replicates whose peak files are associated with this quality metric.",
+            "type": "integer",
+            "minimum": 0
+        },
+        "N_opt": {
+            "title": "Optimal reproducible peaks",
+            "description": "Number of optimal peaks passing reproducibility test between 2 replicates whose peak files are associated with this quality metric.",
+            "type": "integer",
+            "minimum": 0
+        },
+        "N_consv": {
+            "title": "Conservative reproducible peaks",
+            "description": "Number of conservative peaks passing reproducibility test between 2 replicates whose peak files are associated with this quality metric.",
+            "type": "integer",
+            "minimum": 0
+        },
+        "idr_cutoff":{
+            "title": "IDR cutoff",
```

Comment:
Can IDR cutoff be negative?

Reply:
Confirmed with Seth as number between 0 and 1.